infra/discovery/src/apollo/resolvers.js

```diff
@@ -46,7 +72,107 @@ const resolvers = {
         nodeEnv: process.env.NODE_ENV ? process.env.NODE_ENV : 'undefined'
       }
     }
+  },
+  Mutation: {
+    async listingSetScoreTags(_, data, context) {
+      const auth = await authenticate(context.discoveryAuthToken)
+      if (!auth) {
+        throw 'You are not logged in'
```

Comment:
nit: we follow the convention in our codebase to throw an Error object. So like:
throw new Error('You are not logged in')